kibana/connector.py

```diff
@@ -100,14 +116,16 @@ def request(self, method, uri, params=None, data=None, error=True, verbose=True,
             try:
                 response.raise_for_status()
             except requests.exceptions.HTTPError:
+                if response.status_code == 404:
```

Comment:
Because this backports and rather than chase around API compatibility, this wraps general 404 with a specific error message